src/LuaTransformer.ts

```diff
@@ -1797,7 +1797,10 @@ export class LuaTransformer {
             );
         }
 
-        if (ts.isPropertyAccessExpression(expression.left) || ts.isElementAccessExpression(expression.left)) {
+        if (
```

Comment:
This is a strange line break

Reply:
oh nvm I see how it's laid out now, maybe add parentheses around the && statement to indicate order though

Reply:
ok I added parentheses